Atomic/syscontainers.py

```diff
@@ -306,6 +346,19 @@ def get_system_containers(self):
             ret.append(container)
         return ret
 
+    def delete_image(self, image):
+        repo = self._get_ostree_repo()
```

Comment:
are you fine if I add an `allow_no_image` parameter?  I would like that the `delete_image` does not need a time of check to time of use race condition, that can happen if another process deletes the same image between the time the caller check its existence and deletes it.